opengb/seed-core
# Replace outbound email templates with OPEN-branded versions

#### Any background context you want to provide?
OPEN requires customized versions of all of the outbound emails sent by SEED.
This is needed primarily for legal reasons: new users need to be directed
to view the Privacy Policy and Terms & Conditions before they start using SEED.
This is also needed for branding purposes.

#### What's this PR do?
- replace plain-text email templates with customized versions for OPEN
- add HTML email templates, customized for OPEN
- make PASSWORD_RESET_EMAIL configurable as an environment variable,
  in the Dockerized version of SEED
- make PROTOCOL (http or https) configurable via an environment variable
- ensure STATIC_URL asset location is correctly set for new-account and
  added-to-org emails
- make the password-reset, new-account, and added-to-org emails
  send both plain-text and HTML versions of the email
- add OPEN-related logo files
- add a base Django template for the HTML email templates,
  which provides the custom font for each email

#### How should this be manually tested?
1. Set up a new instance of SEED with outbound email capability.
2. Create a new user. Ensure the user's email address is a real one that you have control over.
3. Check that the account-created email is received at the new user's email address. Check that it has custom images, a custom font, and that the text makes sense.
4. Create a new organization.
5. Invite the user you created in step 2 to the new organization.
6. Check that the added-to-organization email is received at the user's email address. Check that it has custom images, a custom font, and that the text makes sense.
7. Log out of your account.
8. Navigate to the forgot-password page.
9. Send a password-reset email to the user with the same email as the one you created in step 2.
10. Check that the password-reset email is received at the user's email address. Check that it has custom images, a custom font, and that the text makes sense.

#### What are the relevant tickets?
N/A

#### Screenshots (if appropriate)
Here are PDFs of the new emails, generated during development.

[New SEED account.pdf](https://github.com/opengb/seed-core/files/5142604/New.SEED.account.pdf)

[Password reset on SEED.pdf](https://github.com/opengb/seed-core/files/5142605/Password.reset.on.SEED.pdf)

[Your SEED account has been added to an organization.pdf](https://github.com/opengb/seed-core/files/5142606/Your.SEED.account.has.been.added.to.an.organization.pdf)
